CoreFindView/test/unit/src/au/gov/asd/tac/constellation/views/find2/components/FindViewTabsNGTest.java

```diff
@@ -77,6 +77,26 @@ public void setUpMethod() throws Exception {
     public void tearDownMethod() throws Exception {
     }
 
+    /**
+     * Test listener for changing selected tab
+     */
+    @Test
+    public void testChangeTab() {
```

Comment:
Was there something wrong with this test? Just noticing all the code in it is commented out

Reply:
Yeah I believe it just wasn't working originally then I forgot about it.